packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/src/Providers/MrwSerializationTypeDefinition.cs

```diff
@@ -27,6 +27,7 @@ namespace Microsoft.Generator.CSharp.ClientModel.Providers
     internal class MrwSerializationTypeDefinition : TypeProvider
     {
         private const string PrivateAdditionalPropertiesPropertyName = "_serializedAdditionalRawData";
+        private const string AdditionalPropertiesPropertyName = "AdditionalProperties";
```

Comment:
I think with this implementation we are treating SARD and AP as two separate things.  I think we should treat them as one thing with the only difference being one is public and one is not.  When it is public it has a few implications like the property will be public and serialization will always emit it vs when its not public serialization will only emit when format is not W.